### ISSUE TITLE
RegionUtils.getRegionsForService(AWSElasticBeanstalk.ENDPOINT_PREFIX) has different respond in version 1.11 and 1.12

### ISSUE THREAD

**Issue by weinick** [aws/aws-sdk-java#2657]
Hi experts, 
I'm using RegionUtils.getRegionsForService(AWSElasticBeanstalk.ENDPOINT_PREFIX) to get the endpoint of Beanstalk in SDK 1.11.480, I found the respond doesn't include all of the regions meanwhile SDK 1.12.84 can get the full region list.

**Comment by weinick** [comment]
Hi @debora-ito thanks for your quick answer, now I know the reason. Since the static file is not updated enough sometimes, I found another possible solution. I found there is another function in V2 SDK which is something like DynamoDbClient.serviceMetadata().regions().forEach(System.out::println). I guess the backend of this function is not based on static file like endpoints.json. So maybe this is a better solution to get the updated region list dynamically?
